apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -326,13 +331,16 @@ protected override DbDataReader ExecuteDbDataReader(CommandBehavior behavior)
                 throw new InvalidOperationException("Unable to open the Spanner connection to the database.");
 
             // Execute the command.
-            var resultset = await GetSpannerTransaction()
-                .ExecuteQueryAsync(CommandText, cancellationToken)
-                .ConfigureAwait(false);
+            var task = GetSpannerTransaction().ExecuteQueryAsync(CommandText, cancellationToken);
```

Comment:
There may be a simpler way to do this - if you need it in more than a couple of places, we should definitely look at abstracting it, whether here or in GAX.